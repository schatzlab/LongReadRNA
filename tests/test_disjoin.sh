cat tests/test_disjoin.tsv | perl scripts/disjoin.pl > test_disjoin.tsv.out
diff test_disjoin.tsv.out tests/test_disjoin.tsv.expected
