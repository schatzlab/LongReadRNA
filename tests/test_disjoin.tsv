chr1	1	5	+
chr1	3	7	+
chr1	4	8	+
chr2	10	15	+
chr2	10	20	+
chr2	17	30	+
chr3	50	60	+
chr3	60	70	+
chr3	65	67	+
chr4	100	110	+
chr4	105	110	+
chr5	200	205	+
chr6	300	300	+
