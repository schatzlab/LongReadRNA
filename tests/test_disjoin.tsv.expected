chr1	1	2	+	+	+
chr1	3	3	+	+	+
chr1	4	5	+	+	+
chr1	6	7	+	+	+
chr1	8	8	+	+	+
chr2	10	15	+	+	+
chr2	16	16	+	+	+
chr2	17	20	+	+	+
chr2	21	30	+	+	+
chr3	50	59	+	+	+
chr3	60	60	+	+	+
chr3	61	64	+	+	+
chr3	65	67	+	+	+
chr3	68	70	+	+	+
chr4	100	104	+	+
chr4	105	110	+	+
chr5	200	205	+
